server/lib/src/idm/oauth2.rs

```diff
@@ -366,8 +366,9 @@ enum OauthRSType {
     Basic {
         authz_secret: String,
         enable_pkce: bool,
+        enable_consent_prompt: bool,
```

Comment:
This became `enable_x` to be consistent with `pkce` - lmk if you prefer `consent_prompt_enable` to match the attribute

Reply:
Sounds good!